packages/mongo/src/db.ts

```diff
@@ -30,8 +35,8 @@ export const initMongoNoIndexes = async () => {
     return;
   }
   const options: MongoClientOptions = {
-    useNewUrlParser: true,
-    useUnifiedTopology: true,
+    // useNewUrlParser: true,
+    // useUnifiedTopology: true,
```

Comment:
This can be safely removed. As [the docs for v4](https://www.mongodb.com/docs/drivers/node/current/whats-new/) state:

> It is longer required to specify `useUnifiedTopology` or `useNewUrlParser`.